client/src/components/rooms/RoomsMeetingExperience.test.tsx

```diff
@@ -82,4 +104,58 @@ describe('RoomsMeetingExperience', () => {
     expect(callComposite.first().props().adapter.getState().displayName?.includes('Virtual appointments User'));
     expect(callComposite.first().props().callInvitationUrl).toBeUndefined;
   });
+
+  it('should render Survey component when postcall is defined and valid', async () => {
```

Comment:
lets say, it('should render Survey component when post call is defined and user is a attendee')
and lets add one more test saying it('should not render Survey component if user is a presenter')